### ISSUE TITLE
[SUGGESTION]: ultra warp ride dungeon

### ISSUE THREAD

**Issue by catlons** [pokeclicker/pokeclicker#5514]
### Current Behavior

the ultra warp ride minigame is unimplemented

### Improved Behavior

a dungeon unlocked after finishing the ultra beast hunt questline that has all of the legendaries up to gen 6 excluding zygarde as the bosses

### Reason for change

there is currently no implementation of this minigame

### Detailed Description

a dungeon including all legendaries up to gen 6 excluding zygarde that has all pokemon available in the ultra sun/ultra moon ultra warp ride minigame including ultra beasts, it could also be added to the eater of light questline but without the legendaries and ultra beasts but with ultra necrozma as a trainer as the boss and be made unavailable afterwards until you finish the ultra beast hunt questline

**Comment by Qwertypop04** [comment]
Why? What would this have to make it worth existing?

**Comment by catlons** [comment]
> Why? What would this have to make it worth existing?
it doesn't exist in the game right now

**Comment by Qwertypop04** [comment]
> it doesn't exist in the game right now
> 

Correct. I'm asking you why it should.
What could it add to the game? A new Pokémon? A new item? We need something to justify it existing. Another dungeon just for the sake of another dungeon isn't good enough, especially in a region like Alola which already has so many

**Comment by catlons** [comment]
> > it doesn't exist in the game right now
> 
> Correct. I'm asking you why it should. What could it add to the game? A new Pokémon? A new item? Something else? We need something to justify it existing. Another dungeon just for the sake of another dungeon isn't good enough, especially in a region like Alola which already has so many

it may be a good way to obtain more legendaries ev's or farm for various item drops

**Comment by catlons** [comment]
it also makes the leadup to ultra necrozma cooler

**Comment by Qwertypop04** [comment]
> it may be a good way to obtain more legendaries ev's or farm for various item drops

I don't think it would be able to do that. Since there are so many legendaries, it would always be more efficient to farm them in their home region, and less expensive too

**Comment by catlons** [comment]
also easier ultra beasts in a more concentrated place

**Comment by Qwertypop04** [comment]
> also easier ultra beasts in a more concentrated place

No. A conscious decision was made to use the SM locations of the UBs and not the the USUM locations (with the exception of the ones introduced in USUM, which would not be in this dungeon anyway)
